src/backend.rs

```diff
@@ -346,7 +392,12 @@ pub fn prologue(ctx: &mut Context, stack_slots: u32) {
 }
 
 pub fn epilogue(ctx: &mut Context) {
-    assert_eq!(ctx.sp_depth, StackDepth(0), "imbalanced pushes and pops detected");
+    // TODO: This doesn't work with stack alignment.
+    // assert_eq!(
+    //     ctx.sp_depth,
+    //     StackDepth(0),
+    //     "imbalanced pushes and pops detected"
+    // );
```

Comment:
I'm not sure how to cleanly fix this one...

Reply:
I'm ok dealing with this later, but eventually we will need to remember how much space we used for stack alignment or other things, which should let us reenable this assert.